meta-sokol-flex-distro/openembedded-layer/recipes-support/lvm2/lvm2_%.bbappend

```diff
@@ -4,16 +4,18 @@
 
 FILESEXTRAPATHS:prepend := "${THISDIR}/${PN}:"
 
-SRC_URI:append = " file://lvm2-mel.service"
+SRC_URI:append = " file://lvm2-flex.service"
 
-PACKAGES += "${PN}-mel"
-SYSTEMD_PACKAGES += "${PN}-mel"
-SYSTEMD_SERVICE:${PN}-mel = "lvm2-mel.service"
-SYSTEMD_AUTO_ENABLE:${PN}-mel = "enable"
-RDEPENDS:${PN} += "${PN}-mel"
+PACKAGES += "${PN}-flex"
+RDEPENDS:${PN} += "${PN}-flex"
+SYSTEMD_PACKAGES += "${PN}-flex"
+SYSTEMD_SERVICE:${PN}-flex = "lvm2-flex.service"
+SYSTEMD_AUTO_ENABLE:${PN}-flex = "enable"
+RREPLACES:${PN}-flex += "${PN}-mel"
+RCONFLICTS:${PN}-flex += "${PN}-mel"
```

Comment:
why do we need to have the rreplaces and conflicts here? we've already made the transition to flex and haven't done a release on ginkgo/kirkstone that would use lvm2-mel

Reply:
It's best practice in Yocto projects to allow for package upgrades to happen. Admittedly we don't do any testing with package feeds, and certainly don't test upgrades across major version, but there's no real cost to allowing it to work. I'm used to considering it when I work on recipes. I don't feel terribly strongly about this, however.

Reply:
It's best practice in Yocto projects to allow for package upgrades to happen. Admittedly we don't do any testing with package feeds, and certainly don't test upgrades across major version, but there's no real cost to allowing it to work.. I'm used to considering it when I work on recipes. I don't feel terribly strongly about this, however.